lib/rucio/tests/test_preparer.py

```diff
@@ -20,6 +20,7 @@
 from rucio.core.replica import add_replicas
 from rucio.core.request import list_transfer_requests_and_source_replicas, set_transfer_limit, list_transfer_limits
 from rucio.core.transfer import get_supported_transfertools
+from rucio.core.rse import REGION
```

Comment:
no, I don't like this one at all. The name 'REGION' is used all over rucio for the "local cache instance". Importing the cache instance from another file without changing it's name is prone to generate confusion.

A correct way to improve this test would be to use the caches_mock fixture. And example: 
https://github.com/rucio/rucio/blob/e2f5808eb265856edba5aa1bf69f66bc2843fdb9/lib/rucio/tests/test_reaper.py#L201

Reply:
I'm a bit confused, I should rename it like `from rucio.core.rse import REGION as REGION_test_preparer` for example or delete this import at all and write 
```
[cache_region] = caches_mock
cache_region.invalidate()
``` 
in `test_preparer_without_and_with_mat` function in `test_preparer.py`?

Reply:
Substitute:
```
@pytest.mark.noparallel(reason='uses preparer')
def test_preparer_without_and_with_mat(db_session, source_rse, dest_rse, mock_request):
    ...
    REGION.invalidate()
```

with:
```
@pytest.mark.noparallel(reason='uses preparer')    
@pytest.mark.parametrize("caches_mock", [{"caches_to_mock": [
    'rucio.core.rse.REGION'
]}], indirect=True)
def test_preparer_without_and_with_mat(db_session, source_rse, dest_rse, mock_request, caches_mock):
    ...
    [cache_region] = caches_mock
    cache_region.invalidate()
```

This way, the test will not pollute the cache for other tests.